CMakeLists.txt

```diff
@@ -540,7 +545,7 @@ if(QUDA_QMP)
       GIT_SHALLOW YES
       PREFIX usqcd
       CONFIGURE_COMMAND CC=${MPI_C_COMPILER} CXX=${MPI_CXX_COMPILER} <SOURCE_DIR>/configure "INSTALL=${INSTALL_EXE} -C"
-                        --with-qmp-comms-type=MPI --prefix=<INSTALL_DIR>
+                        "CFLAGS=-Wall -O3 -std=c99 -g" --with-qmp-comms-type=MPI --prefix=<INSTALL_DIR>
```

Comment:
What's the motivation of `-g` by default here?

Reply:
Ease of debugging when needed, combined with the generally negligible performance overheads of `-g`. (It's also what I've used in the past, plus what Carleton gave me for reference, though those aren't necessarily good long term reasons.)

I can add the logic to only include `-g` for non-`RELEASE` builds of QUDA, if that's preferred.

Reply:
That would be preferred, I think yes, if it's easy enough to add.  Thx